src/main/kotlin/org/rekotlin/Store.kt

```diff
@@ -54,7 +54,7 @@ class Store<State: StateType> (
                 middleware(dispatch, getState)(dispatchFunction)
             })
 
-    val subscriptions: MutableList<SubscriptionBox<State, Any>> = mutableListOf()
+    val subscriptions: java.util.concurrent.CopyOnWriteArrayList<SubscriptionBox<State, Any>> = CopyOnWriteArrayList()
```

Comment:
you should probably use an import statement instead of the fully qualified name

Reply:
sure